bindings/core-utils.i

```diff
@@ -157,6 +157,9 @@ gchar *gnc_locale_name (void);
 
 }
 
+%rename ("gnc:ngettext") ngettext;
+extern const char* ngettext (const char *msgid1, const char *msgid2,
+                             unsigned long int n);
```

Comment:
`(define (NG_ msgid1 msgid2) (gettext msgid1 msgid2));` seems missing, if you  want to use that abbreviated form.